### DISCUSSION TITLE
Query : Device value of type Array

### DISCUSSION THREAD

**Discussion by Zehir** [deconz-community/node-red-contrib-deconz#172] [Ideas]
Method to compare when device have value in array like `devicemembership`.
Possible operator :
* Contain one (Ex get all groups who have the light X)
* Contain all (Ex get all groups who have the ligths X, Y and maybe more)
* Equal (Ex get all groups who have only the lights X and Y)

**Comment by TheNON75** [comment]
This sounds great
